webapp/src/store/boards.ts

```diff
@@ -28,17 +28,17 @@ export const fetchBoardMembers = createAsyncThunk(
     async ({teamId, boardId}: {teamId: string, boardId: string}, thunkAPI: any) => {
         const members = await client.getBoardMembers(teamId, boardId)
         const users = [] as IUser[]
+        const userArr = [] as string[]
 
         /* eslint-disable no-await-in-loop */
         for (const member of members) {
-            // TODO #2968 we should fetch this in bulk
-            const user = await client.getUser(member.userId)
-            if (user) {
-                users.push(user)
-            }
+            userArr.push(member.userId)
```

Comment:
Syntactic sugar - you can simply do this to make it look (subjectively) neater-

```js
const userArr = members.map((member) => member.userId)
```

Reply:
Done